src/main/java/core/DataProvidersBoard.java

```diff
@@ -0,0 +1,76 @@
+package core;
+
+import org.testng.annotations.DataProvider;
+
+public class DataProvidersBoard {
+
+    @DataProvider(name = "correctColors")
+    public static Object[][] provideCorrectColors() {
+        return new Object[][] {
+            {"orange"},
```

Comment:
Data providers should be constants or enums

Reply:
Fixed.